cmd/avalanche.go

```diff
@@ -13,6 +14,26 @@ import (
 	kingpin "gopkg.in/alecthomas/kingpin.v2"
 )
 
+type ignoredIntervals map[string]bool
+
+func (i ignoredIntervals) Set(value string) error {
+	fields := strings.Split(value, ",")
+	for _, f := range fields {
+		i[f] = true
+	}
+	return nil
+}
+
+func (i ignoredIntervals) String() string {
```

Comment:
this unused function is just to allow `ignoreIntervals` to be used as an input to `kingpin.Settings.SetValue`?